docs/data/toolpad/getting-started/overview.md

```diff
@@ -0,0 +1,34 @@
+# Overview
+
+<p class="description">
+Learn core concepts of the Toolpad app and you are ready to ship your application in minutes!
+</p>
+
+## What is Toolpad?
```

Comment:
Let's stick more closely to the formatting of the existing [Overview pages](https://mui.com/base/getting-started/overview/) for the sake of consistency.

```suggestion
## Introduction
```